frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -145,15 +145,22 @@ export const PublicFormProvider = ({
     featureFlags.turnstile,
     false,
   )
+  enum CaptchaTypes {
```

Comment:
Artifact from refactoring? 
Seems like this had been declared in `src/app/services/captcha/captcha.types.ts`.

Reply:
Wanted to separate the usage of types from the frontend and backend codebase. Refactored and moved the declaration to the shared folder as discussed.